docs/conf.py

```diff
@@ -67,7 +67,7 @@
 # General information about the project.
 project = u'Autoprotocol'
 year = datetime.now().year
-copyright = u'%d, Transcriptic Inc.' % year
+copyright = u'%d, The Autoprotocol Development Team, see AUTHORS for more details.' % year
```

Comment:
assuming this change is allowed since for now it's all really the same entity and there's a clarification made in AUTHORS about it?

Reply:
yup, that's my working assumption here